### ISSUE TITLE
Property 'axios' does not exist on type 'Transport'

### ISSUE THREAD

**Issue by elixiao** [feathersjs/feathers-rest#116]
I want to use axios as my ajax tool:

```
const app = feathers()
    .configure(rest('http://baseUrl').axios(axios))
```

But there is an error: `Property 'axios' does not exist on type 'Transport' `, I check the `client.d.ts `as follows:

```
interface Transport {
    jquery: Handler;
    superagent: Handler;
    request: Handler;
    fetch: Handler;
  }
```
find there is no axios here.

**Comment by daffl** [comment]
It has been added in #98 in [v1.6.0](https://github.com/feathersjs/feathers-rest/blob/master/CHANGELOG.md#v160-2016-12-31). You have to make sure that `npm ls feathers-rest` shows version 1.6.0 or later.

**Comment by elixiao** [comment]
`npm ls feathers-rest` shows:
```
├─┬ feathers-client@2.2.0
│ └── feathers-rest@1.7.2  deduped
└── feathers-rest@1.7.2
```

the file `client.d.ts` has no `axios` support

**Comment by eddyystop** [comment]
Axios support exists, but TypeScript definitions are a Feathers TypeScript community effort contributed by TS users. Feel free to submit a PR with the definition.

**Comment by daffl** [comment]
This has been fixed via #117 and now released as v1.7.3 (a little late but from I did it from an airplane 😉).
